CorsixTH/Lua/game_ui.lua

```diff
@@ -1099,8 +1099,10 @@ tutorial_phases = {
           {_S.introduction_texts["level15"]},
           {_S.introduction_texts["level16"]},
           {_S.introduction_texts["level17"]},
-          {_S.introduction_texts["level1"]},
         }
+        if TheApp.world.map.level_number ~= 1 then
+          table.insert(texts, _S.introduction_texts["level1"])
+        end
```

Comment:
Should we also cover the demo's tutorial? Or does the demo one make more logical sense to come after?

Reply:
Yes, both of them before is better.